py/dml/toplevel.py

```diff
@@ -178,6 +178,30 @@ def add_constant(name, expr):
 bidi_re = re.compile(
     '[\u2066-\u2069\u202a-\u202e]')
 
+pragma_re = re.compile(r'^\s*///\s*(\S+)\s*(?:\s(\S.*))?$')
+pragma_coverity_data_re = re.compile(r'(\S+)\s*(?:\s(\S.*))?$')
+
+test_pragmas = {'WARNING', 'ERROR', 'SCAN-FOR-TAGS', 'DMLC-FLAG', 'CC-FLAG',
+                'GREP', 'INSTANTIATE-MANUALLY', 'COMPILE-ONLY', 'NO-CC'}
+
+nowarn_test_pragmas = os.getenv('DMLC_NOWARN_TEST_PRAGMAS')
```

Comment:
or just delegate enumeration to tests, `nowarn_pragmas = os.getenv['DMLC_NOWARN_TEST_PRAGMAS'].split(':')`

(and these two variables are unused, ideally move to other PR)

Anyway not blocking.

Reply:
No longer relevant with the new syntax.